publish-app-do/index.md

```diff
@@ -0,0 +1,458 @@
+      --log_level string    The logging level (trace|debug|info|warn|error|fatal|panic) (default "info")
+      --trace               print out full stack trace on errors
+
+Use "digitaloceand [command] --help" for more information about a command.
+```
+
+## Create a Systemd process
+
+The `starport serve` command helps installing, building, and configuring your app for a new blockchain. You received tokens on newly created accounts.
+
+This new app is useful only for development purposes and is not intended to be run for a long time in production. 
+
+For the app to run for a longer time, you need a `systemd` process.
+
+Create the log files and a `systemd` file to manage your process so that you can log off from the terminal and be confident the app keeps running even when the server restarts.
+
+```bash
+sudo mkdir -p /var/log/digitaloceand
+sudo touch /var/log/digitaloceand/digitaloceand.log
+sudo touch /var/log/digitaloceand/digitaloceand_error.log
+sudo touch /etc/systemd/system/digitaloceand.service
+```
+
+Use nano to edit the systemd service file:
+
+```bash
+sudo nano /etc/systemd/system/digitaloceand.service
+```
+
+Add your application details to the file:
+
+```bash
+[Unit]
+Description=digitalocean daemon
+After=network-online.target
+[Service]
+User=appuser
+ExecStart=/home/appuser/go/bin/digitaloceand start --home=/home/appuser/.digitalocean
+WorkingDirectory=/home/appuser/go/bin
+StandardOutput=file:/var/log/digitaloceand/digitaloceand.log
+StandardError=file:/var/log/digitaloceand/digitaloceand_error.log
+Restart=always
+RestartSec=3
+LimitNOFILE=4096
+[Install]
+WantedBy=multi-user.target
+```
+
+To close and save the file, press Ctrl+x. Then type `Y` and press Enter. 
+
+The next step is to enable the file so the app is available even after a system start:
+
+```bash
+sudo systemctl enable digitaloceand.service
+```
+
+Now start the process:
+
+```bash
+sudo systemctl start digitaloceand.service
+```
+
+Check the logs with:
+
+```bash
+sudo journalctl -u digitaloceand -f
+```
+
+Your app is now running on DigitalOcean.
+
+## Connect your local running chain to the published chain
+
+To connect your local running chain with the published app, you need to:
+
+- Find the node you just created with the IP address
+- Download the genesis file
+- Configure your app to connect to the published node
+
+On your DigitalOcean droplet, find the hash of the node by typing:
+
+```bash
+digitaloceand tendermint show-node-id
+# For example, 85ef06cb86aa501cceb9ed0a497a02503f5aa57f
+```
+
+Each node has a different node id. 
+
+At a terminal window back on your local machine, initialize your blockchain with:
+
+```bash
+digitaloceand init localnode
+```
+
+Next, download the `genesis.json` file from your node:
+
+```bash
+scp appuser@<IP address>:/home/appuser/.digitalocean/config/genesis.json ~/.digitalocean/config/
+```
+
+Then add the published node ID and IP into the configuration file.
```

Comment:
```suggestion
Then add the published node ID and IP into the configuration file on your local machine:
```

Reply:
I got a little lost here too, maybe we should clarify on the local machine (why do we do this? give a little context would be helpful)